nifi-nar-bundles/nifi-prometheus-bundle/nifi-prometheus-reporting-task/pom.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file mixes tab and spaces indentation, please be consistent. I think in most xml files in the project we are using spaces.